lib/screens/create_task_screen.dart

```diff
@@ -42,12 +47,59 @@ class _CreateTaskScreenState extends State<CreateTaskScreen> {
               style: MainStyles.kBlackColor1W700(26.0),
             ),
             centerTitle: true,
+            leading: InkWell(
+              onTap: () async {
+                dateTime = await showOmniDateTimePicker(
+                  context: context,
+                  initialDate: DateTime.now(),
+                  firstDate: DateTime(1600).subtract(const Duration(days: 3652)),
+                  lastDate: DateTime.now().add(
+                    const Duration(days: 3652),
+                  ),
+                  is24HourMode: false,
+                  isShowSeconds: false,
+                  minutesInterval: 1,
+                  secondsInterval: 1,
+                  borderRadius: const BorderRadius.all(Radius.circular(16)),
+                  constraints: const BoxConstraints(maxWidth: 350, maxHeight: 650),
+                  transitionBuilder: (context, anim1, anim2, child) {
+                    return FadeTransition(
+                      opacity: anim1.drive(
+                        Tween(begin: 0, end: 1),
+                      ),
+                      child: child,
+                    );
+                  },
+                  transitionDuration: const Duration(milliseconds: 200),
+                  barrierDismissible: true,
+                  selectableDayPredicate: (dateTime) {
+                    if (dateTime == DateTime(2023, 2, 25)) {
+                      return false;
+                    } else {
+                      return true;
+                    }
```

Comment:
1. When you have a case like this, don't write if/else. Write return dateTime != DateTime(2023, 2, 25)

2. What is 2023/2/25? Why specifically this date?